pht_federated/aggregator/services/discovery/object_list_transform.py

```diff
@@ -31,15 +29,16 @@ def object_to_list(dataset_statistics_object: DatasetStatistics) -> list:
     return object_list_int
 
 
-def list_to_object(dataset_statistics_list: list, dataset_statistics_object: DatasetStatistics) -> DatasetStatistics:
+def list_to_object(
+    dataset_statistics_list: list, dataset_statistics_object: DatasetStatistics
+) -> DatasetStatistics:
     """
     Transforms a python list of integers that contain all masked numerical values back to a DatasetStatistics object
     :param dataset_statistics_list: list
     :param dataset_statistics_object: DatasetStatistics
     :return: DatasetStatistics
     """
     stats_dict = dataset_statistics_object.dict()
-    #print("Dataset Statistics Object : {}".format(stats_dict))
 
     if "item_count" in stats_dict:
         stats_dict["item_count"] = dataset_statistics_list[0]
```

Comment:
whats going on with the [2:]? why is the input list modified in place?

Reply:
because "item_count" and "feature_count" not of interest further in the function?